### ISSUE TITLE
Incorrect Render with Column Type Date

### ISSUE THREAD

**Issue by coderazzi** [coderazzi/tablefilter-swing#61]
(Original issue 15 created by None on 2011-05-24T23:03:12+00:00)

```
What steps will reproduce the problem?
1. Using a Render for a datatype DATE, with format 'dd/MM/yyyy'
2. Returning an Object.class in getColumnClass 

What is the expected output? What do you see instead?
I wish the combobox show me the date in the render format, but the combobox show me
the date and the time

What version of the product are you using? On what operating system?
TableFilter 4.1.3, Windows XP Sp2, Swingx 1.6.2

Please provide any additional information below.


```

Original issue reported on code.google.com by `sanedrak` on 2011-05-24 23:03:12

**Comment by coderazzi** [comment]
(Original issue 15 last updated on 2015-04-26T00:24:16.370398+00:00)

(Issue automaticaly closed due to status in Bitbucket: resolved)

**Comment by coderazzi** [comment]
(Original comment by {u'display_name': u'coderazzi coderazzi', u'account_id': u'557058:790eb198-2ae4-49b4-b920-8d654aa86b3f'} on 2011-05-24T23:34:24+00:00)

```
Confirmed to be a regression error; any column with a provided Format will behave incorrectly
when a choice is selected on the filter editor combobox.
```

Original issue reported on code.google.com by `coderazzi` on 2011-05-24 23:34:24

**Comment by coderazzi** [comment]
(Original comment by {u'display_name': u'coderazzi coderazzi', u'account_id': u'557058:790eb198-2ae4-49b4-b920-8d654aa86b3f'} on 2011-05-25T22:53:11+00:00)

```
Solved on release 4.1.4
```

Original issue reported on code.google.com by `coderazzi` on 2011-05-25 22:53:11

**Comment by coderazzi** [comment]
(Original comment by {u'display_name': u'coderazzi coderazzi', u'account_id': u'557058:790eb198-2ae4-49b4-b920-8d654aa86b3f'} on 2015-04-26T00:24:16.378119+00:00)

Already resolved (previously, before migrating to BitBucket)
